src/libs/ReportUtils.ts

```diff
@@ -4287,7 +4287,7 @@ function shouldDisableRename(report: OnyxEntry<Report>, policy: OnyxEntry<Policy
 /**
  * @param policy - the workspace the report is on, null if the user isn't a member of the workspace
  */
-function canEditWriteCapability(report: OnyxEntry<Report>, policy: OnyxEntry<Policy>): boolean {
+function canEditWriteCapability(report: OnyxEntry<Report>, policy: OnyxEntry<Policy> | undefined): boolean {
```

Comment:
The comment states that policy could be null if user isn't a member. Let's use null and undo the util function changes (same applies to other similar changes)

Reply:
@fabioh8010 What do you think about it ?